src/particles/transformation/ToFixedT.H

```diff
@@ -59,10 +59,13 @@ namespace impactx::transformation
             amrex::ParticleReal const y = p.pos(RealAoS::y);
             amrex::ParticleReal const t = p.pos(RealAoS::t);
 
+            // small tolerance to avoid NaN for pz<0:
+            amrex::ParticleReal const tol = 1.0e-8_prt;
```

Comment:
Interesting, can you elaborate on this or add an inline comment how we chose this tolerance? I think I do not understand just yet :)

Reply:
This value is rather arbitrary, and "tolerance" is probably not the best name for this constant.  It must be positive, but small enough that it has a near-negligible effect on the particle orbit.  It is needed because p_z cannot be equal to zero.  If it appears that this will happen, the p_z is set to this value to avoid division by zero or other NaN errors.